grpc/src/main/proto/grpc/event_service.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+package conductor.grpc;
+
+import "google/protobuf/empty.proto";
+import "model/eventhandler.proto";
+
+option java_package = "com.netflix.conductor.grpc";
+option java_outer_classname = "EventServicePb";
+option go_package = "github.com/netflix/conductor/client/gogrpc/conductor/grpc";
+
+service EventService {
+    // POST /
+    rpc AddEventHandler(conductor.proto.EventHandler) returns (google.protobuf.Empty);
```

Comment:
Always introduce Request and Response wrappers for all gRPC services so request/response can be extended in the future without breaking compatibility.

Reply:
That's a good point. I'll get to this today.